client/starwhale/utils/venv.py

```diff
@@ -875,20 +875,20 @@ def install_starwhale(
     force: bool = False,
     configs: _ConfigsT = None,
 ) -> None:
-    if version == "" or version == SW_DEV_DUMMY_VERSION:
-        version = get_downloadable_sw_version()
-
-    req = SW_PYPI_PKG_NAME
-    if version:
-        req = f"{req}=={version}"
-
     _existed = check_user_python_pkg_exists(
         str(prefix_path / "bin" / "python3"), SW_PYPI_PKG_NAME
     )
     if _existed and not force:
         console.info(f"{SW_PYPI_PKG_NAME} has already be installed at {prefix_path}")
         return
 
+    if version == "" or version == SW_DEV_DUMMY_VERSION:
+        version = get_downloadable_sw_version()
+
+    req = [SW_PYPI_PKG_NAME]
```

Comment:
req is not needed to change type.

Reply:
There will be an error `starwhale/utils/venv.py: error: Incompatible types in assignment (expression has type "List[str]", variable has type "str")` when lint with mypy

Reply:
fixed